src/ontology/mondo-edit.obo

```diff
@@ -149374,6 +149384,7 @@ synonym: "spondylocostal dysostosis caused by mutation in TBX6" EXACT [MONDO:des
 synonym: "spondylocostal dysostosis type 5" EXACT [MONDORULE:1, OMIM:122600]
 synonym: "spondylocostal dysplasia" RELATED [OMIM:122600]
 synonym: "spondylothoracic dysostosis" EXACT DEPRECATED [https://rarediseases.org/rare-diseases/jarcho-levin-syndrome/, OMIM:122600]
+synonym: "Spondylothoracic Dysplasia" EXACT [NORD:1915]
 synonym: "Spondylothoracic Dysplasia" RELATED [NORD:1915]
```

Comment:
This synonym is for MONDO:0007389 'spondylocostal dysostosis 5'. While 'spondylocostal dysostosis 5' and Spondylothoracic Dysplasia share some features, their genetic mechanisms are different. 
- [https://omim.org/entry/122600](https://omim.org/entry/122600) lists these alternative titles: SPONDYLOCOSTAL DYSPLASIA and SPONDYLOTHORACIC DYSOSTOSIS
- [NORD:1915](https://rarediseases.org/rare-diseases/spondylothoracic-dysostosis/) states "There is significant confusion in the medical literature regarding names for spondylothoracic dysplasia. For years, this disorder and a similar disorder, spondylocostal dysplasia (were considered the same disorder and referred to as Jarcho-Levin syndrome. Researchers now know that these disorders are separate entities with different causes and associated malformations."

Based on NORD it seems that this synonym should be related vs. exact and perhaps should not even be listed as a synonym for 'spondylocostal dysostosis 5'.